components/fxa-client/src/http_client.rs

```diff
@@ -91,6 +91,11 @@ pub trait FxAClient {
         update: DeviceUpdateRequest<'_>,
     ) -> Result<UpdateDeviceResponse>;
     fn destroy_device(&self, config: &Config, refresh_token: &str, id: &str) -> Result<()>;
+    fn attached_clients(
+        &self,
+        config: &Config,
+        refresh_token: &str,
```

Comment:
the signature here still says `refresh_token` (which I guess doesn't cause compilation errors because it's the correct *type* just with the wrong name).